ARM-wvd-templates/AddVirtualMachinesToHostPool/AddVirtualMachinesTemplate.json

```diff
@@ -1,502 +1,576 @@
+        "description": "The token of the host pool where the session hosts will be added."
+      }
+    },
+    "hostpoolResourceGroup": {
+      "type": "string",
+      "metadata": {
+        "description": "The resource group of the host pool to be updated. Used when the host pool was created empty."
+      },
+      "defaultValue": ""
+    },
+    "hostpoolLocation": {
+      "type": "string",
+      "metadata": {
+        "description": "The location of the host pool to be updated. Used when the host pool was created empty."
+      },
+      "defaultValue": ""
+    },
+    "hostpoolProperties": {
+      "type": "object",
+      "metadata": {
+        "description": "The properties of the Hostpool to be updated. Used when the host pool was created empty."
+      },
+      "defaultValue": {}
+    },
+    "vmTemplate": {
+      "type": "string",
+      "metadata": {
+        "description": "The host pool VM template. Used when the host pool was created empty."
+      },
+      "defaultValue": ""
+    },
+    "administratorAccountUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "A username in the domain that has privileges to join the session hosts to the domain. For example, 'vmjoiner@contoso.com'."
+      },
+      "defaultValue": ""
+    },
+    "administratorAccountPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "The password that corresponds to the existing domain username."
+      },
+      "defaultValue": ""
+    },
+    "vmAdministratorAccountUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "A username to be used as the virtual machine administrator account. The vmAdministratorAccountUsername and  vmAdministratorAccountPassword parameters must both be provided. Otherwise, domain administrator credentials provided by administratorAccountUsername and administratorAccountPassword will be used."
+      },
+      "defaultValue": ""
+    },
+    "vmAdministratorAccountPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "The password associated with the virtual machine administrator account. The vmAdministratorAccountUsername and  vmAdministratorAccountPassword parameters must both be provided. Otherwise, domain administrator credentials provided by administratorAccountUsername and administratorAccountPassword will be used."
+      },
+      "defaultValue": ""
+    },
+    "availabilityOption": {
+      "type": "string",
+      "metadata": {
+        "description": "Select the availability options for the VMs."
+      },
+      "defaultValue": "None",
+      "allowedValues": [
+        "None",
+        "AvailabilitySet",
+        "AvailabilityZone"
+      ]
+    },
+    "availabilitySetName": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of avaiability set to be used when create the VMs."
+      },
+      "defaultValue": ""
+    },
+    "createAvailabilitySet": {

```

Comment:
I think this can be removed because the availabilityOption has a value of "None" that can be used.

Reply:
we need this when customer select create new avail set.